app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    if people:
+        for member in people:
+            person_list.append(Person(member["name"], member["age"]))
+
+        for member in people:
+            if "husband" in member.keys():
```

Comment:
```suggestion
            if "husband" in member:
```
It's more clear way to check key presence